utils/kata-manager.sh

```diff
@@ -327,7 +327,7 @@ check_deps()
 		debian|ubuntu) sudo apt-get -y install $packages ;;
 		fedora) sudo dnf -y install $packages ;;
 		opensuse*|sles) sudo zypper install -y $packages ;;
-		*) die "Unsupported distro: $ID"
+		*) die "Unsupported distro: $ID, install $packages manually and re-run"
```

Comment:
Just a tip from the user's perspective, and I know it's been there before your PR. :-)

Drop the "Unsupported distro: " part of the message.  It would make a user run away thinking the whole kata-containers is not supported on that distro, which is not the case.

Reply:
Done!